frontend/src/services/fundingAgreementService.js

```diff
@@ -47,19 +52,7 @@ export default {
     }
   },
 
-  // A facility can have only 1 Active funding agreement
-  async getActiveFundingAgreementByFacilityId(facilityId) {
-    try {
-      if (!facilityId) return
-      const response = await ApiService.apiAxios.get(`${ApiRoutes.FUNDING_AGREEMENTS}?facilityId=${facilityId}&stateCode=${CRM_STATE_CODES.ACTIVE}`)
-      return response?.data[0]
-    } catch (error) {
-      console.log(`Failed to get the active funding agreement by facility id - ${error}`)
-      throw error
-    }
-  },
-
-  // A facility can have only 1 Active funding agreement
+  //Node.js will order FA's so newest one (newest MOD agreement) will always be first
   async getActiveFundingAgreementByFacilityIdAndStatus(facilityId, statusCode) {
```

Comment:
This function is used in SignFundingPopup. Do we need to update this function to return all FAs instead of just the 1st one? As you mentioned in the meeting, there could be multiple FA mod with signature pending?

Reply:
I was on the fence about this one.. It sounded like there "shouldn't be" the case of multiple FA's to sign.. However even if there is a FA to sign, and it is not the most recent one - this will still return it. I can def change it to support that... what do you think?

Reply:
I think yours looks good because we only show max 3 FAs in the popup. also, once they go to Funding Agreement Tab, they can see the full list of all Sig Pending FAs. we should be good.